phpcs.xml.dist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```

Comment:
Let's rename and update (still removing outdated VIP rules) from `phpcs.ruleset.xml`.

Reply:
Looks like this file isn't necessary either since `phpcs.ruleset.xml` covers everything. Should I remove this from the commit too?